pages/common/tslint.md

```diff
@@ -0,0 +1,19 @@
+# tslint
+
+> A pluggable linting utility for TypeScript.
+
+- Create tslint config:
+
+`tslint --init`
+
+- Lint on files:
```

Comment:
Let's make the same change as `xo` and this should be good to go.

Lint on a given set of files.